Products/ZenEvents/EventClass.py

```diff
@@ -67,52 +71,56 @@ class EventClass(EventClassPropertyMixin, Organizer, ManagedEntity, ZenPackable)
     default_catalog = "eventClassSearch"
 
     _relations = ZenPackable._relations + (
-        ("instances", ToManyCont(ToOne, "Products.ZenEvents.EventClassInst", "eventClass")),
-        )
-
-
-    _properties = Organizer._properties + \
-                  EventClassPropertyMixin._properties
+        (
+            "instances",
+            ToManyCont(
+                ToOne,
+                "Products.ZenEvents.EventClassInst",
+                "eventClass"
+            )
+        ),
+    )
 
+    _properties = Organizer._properties + EventClassPropertyMixin._properties
 
     # Screen action bindings (and tab definitions)
     factory_type_information = (
         {
-            'id'             : 'EventClass',
-            'meta_type'      : 'EventClass',
-            'description'    : """Base class for all event classes""",
-            'icon'           : 'EventClass.gif',
-            'product'        : 'ZenEvents',
-            'factory'        : 'manage_addEventClass',
-            'immediate_view' : 'eventClassStatus',
-            'actions'        :
-            (
-                { 'id'            : 'classes'
-                , 'name'          : 'Classes'
-                , 'action'        : 'eventClassStatus'
-                , 'permissions'   : (
-                  Permissions.view,)
+            'id': 'EventClass',
+            'meta_type': 'EventClass',
+            'description': """Base class for all event classes""",
+            'icon': 'EventClass.gif',
+            'product': 'ZenEvents',
+            'factory': 'manage_addEventClass',
+            'immediate_view': 'eventClassStatus',
```

Comment:
For readability purposes, having a stack of key/value pairs should probably be formatted into columns, e.g.
```
            'id':             'EventClass',
            'meta_type':      'EventClass',
            'description':    'Base class for all event classes',
            'icon':           'EventClass.gif',
            'product':        'ZenEvents',
            'factory':        'manage_addEventClass',
            'immediate_view': 'eventClassStatus',
```
and using triple quotes (`"""`) is just a syntax for writing multi-line strings.  No need to use them if you have only a single line of text.
Note: The colon (`:`) is still next to the key name.  Adding spaces between the colon and the value hasn't been flagged by the PEP8 checkers I've used.

Reply:
+